src/PreCICE.jl

```diff
@@ -64,227 +85,1031 @@ function createSolverInterfaceWithCommunicator(participantName::String,
 end
 
 
+@doc """
+
+    initialize()::Float64
+
+Fully initializes preCICE.
+This function handles:
+- Parallel communication to the coupling partner/s is setup.
+- Meshes are exchanged between coupling partners and the parallel partitions are created.
+- **Serial Coupling Scheme:** If the solver is not starting the simulation, coupling data is received from the coupling partner's first computation.
+
+Return the maximum length of first timestep to be computed by the solver.
```

Comment:
Can we state the data type of the return argument just like we can do for input arguments? That would be helpful for the users

Reply:
The return type is given by `initialize()::Float64` in the signature, do you think this is enough or should we also put this in the text?

Reply:
Okay, so the data type next to the function name is the return type. Got it, looks good, lets stay with this :+1: